.github/workflows/auto-merge-release.yml

```
@@ -0,0 +1,18 @@
+name: Auto-merge Release Please PRs
+
+on:
+  pull_request:
+    types: [labeled]
+
+jobs:
+  enable-auto-merge:
+    if: contains(github.event.pull_request.labels.*.name, 'autorelease: pending')
```

actionlint (1.7.9)

could not parse as YAML: mapping values are not allowed in this context

(syntax-check)